backend/molgenis-emx2/src/main/java/org/molgenis/emx2/SchemaMetadata.java

```diff
@@ -115,7 +114,7 @@ public List<TableMetadata> getTables() {
     for (String tableName : getTableNames()) {
       result.add(getTableMetadata(tableName));
     }
-    TableSort.sortTableByDependency(result);
+    // TableSort.sortTableByDependency(result);
```

Comment:
what is the impact of not sorting tables?

Reply:
nothing, this was to ensure tests would not fail because of deterministic order. In practices not needed.

Reply:
delete instead of commenting out

Reply:
removed